task_sdk/src/airflow/sdk/definitions/_internal/abstractoperator.py

```diff
@@ -136,6 +140,81 @@ class AbstractOperator(Templater, DAGNode):
         )
     )
 
+    def unmap(self, resolve) -> BaseOperator:
+        """
+        Get the "normal" operator from current abstract operator.
+
+        MappedOperator uses this to unmap itself based on the map index. A non-
+        mapped operator (i.e. BaseOperator subclass) simply returns itself.
+
+        :meta private:
+        """
+        raise NotImplementedError()
+
+    @cached_property
+    def global_operator_extra_link_dict(self) -> dict[str, Any]:
+        """Returns dictionary of all global extra links."""
+        from airflow import plugins_manager
+
+        plugins_manager.initialize_extra_operators_links_plugins()
+        if plugins_manager.global_operator_extra_links is None:
+            raise AirflowException("Can't load operators")
+        return {link.name: link for link in plugins_manager.global_operator_extra_links}
```

Comment:
I wonder if we should keep global operator links defined /loaded from the webserver instead of in the task sdk

(I think yes, that way we don't need to use plugin manager in task SDK.

Reply:
Not a bad idea at all. Let me track this in the latest related comment